platform_umbrella/apps/verify/test/support/kind_install_worker.ex

```diff
@@ -0,0 +1,202 @@
+defmodule Verify.KindInstallWorker do
```

Comment:
I moved this in a separate commit which should have made the diff easier but you can look at the individual commits.